config/envs/default.py

```diff
@@ -105,11 +106,21 @@ class DefaultConfig(object):
     APPLICANT_FRONTEND_COOKIE_POLICY_URL = (
         APPLICANT_FRONTEND_HOST + "/cookie_policy"
     )
+    APPLICANT_FRONTEND_CONTACT_US_URL = (
+            APPLICANT_FRONTEND_HOST + "/contact_us"
+    )
+    APPLICATION_ALL_QUESTIONS_URL = (
+            APPLICANT_FRONTEND_HOST + "/cof-r2w2-all-questions"
+    )
 
     # Fund store service
     FUND_STORE_API_HOST = CommonConfig.FUND_STORE_API_HOST
     FUND_STORE_FUND_ENDPOINT = CommonConfig.FUND_ENDPOINT
 
+    GET_ROUND_DATA_FOR_FUND_ENDPOINT = (
+            FUND_STORE_API_HOST + "/funds/{fund_id}/rounds/{round_id}"
```

Comment:
Already defined in `commonconfig.py` so change to use that https://github.com/communitiesuk/funding-service-design-utils/blob/6a7603ad45667972159e379c044fcf9a61905162/fsd_utils/config/commonconfig.py#L77
```suggestion
            FUND_STORE_API_HOST + CommonConfig.ROUND_ENDPOINT
```